cli/tests/integration/lsp_tests.rs

```diff
@@ -11,21 +11,21 @@ use std::fs;
 use std::process::Stdio;
 use test_util::deno_cmd_with_deno_dir;
 use test_util::env_vars_for_npm_tests;
-use test_util::lsp::LspClientBuilder;
 use test_util::testdata_path;
 use test_util::TestContextBuilder;
 use tower_lsp::lsp_types as lsp;
 
 #[test]
 fn lsp_startup_shutdown() {
-  let mut client = LspClientBuilder::new().build();
+  let context = TestContextBuilder::new().use_temp_cwd().build();
```

Comment:
I think this should be the default actually and it should opt out of using a temp cwd. I will look into this in a future PR and it would allow me to revert a lot of this.